plugins/aws_sts/test/samson_aws_sts/samson_plugin_test.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require_relative '../test_helper'
+
+SingleCov.covered! uncovered: 2
+
+describe SamsonAwsSts do
+  let(:stage) { stages(:test_staging) }
+
+  describe '.set_env_vars' do
+    it 'sets values against ENV' do
+      %w(
```

Comment:
use the with_env helper for things like this ... can do `with_env FOO: nil` to enforce things are unset